### PR TITLE
Update MapboxNavigation to v2.2.0

### PR DESCRIPTION
**Pull request by ShanMa1991**
This PR is to update `MapboxNavigation` to `v2.2.0`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ To install the MapboxNavigation framework using [Carthage](https://github.com/Ca
 1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
    ```cartfile
    # Latest stable release
-   github "mapbox/mapbox-navigation-ios" ~> 2.0
+   github "mapbox/mapbox-navigation-ios" ~> 2.2
    # Latest prerelease
    github "mapbox/mapbox-navigation-ios" "v2.2.0-rc.1"
```
Comment (MaximAlien) [inline README.md:106]: We'll have to update this part when `v2.3.0-alpha.1` is out.

File: README.md [hunk 2]
```diff
@@ -72,8 +72,8 @@ To install the MapboxNavigation framework using [CocoaPods](https://cocoapods.or
 1. Create a [Podfile](https://guides.cocoapods.org/syntax/podfile.html) with the following specification:
    ```ruby
    # Latest stable release
-   pod 'MapboxCoreNavigation', '~> 2.0'
-   pod 'MapboxNavigation', '~> 2.0'
+   pod 'MapboxCoreNavigation', '~> 2.2'
+   pod 'MapboxNavigation', '~> 2.2'
    # Latest prerelease
    pod 'MapboxCoreNavigation', :git => 'https://github.com/mapbox/mapbox-navigation-ios.git', :tag => 'v2.2.0-rc.1'
```
Comment (MaximAlien) [inline README.md:78]: Same here.

**Review by MaximAlien — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ShanMa1991) [reply README.md:106]: So we don't update it right now, but wait for the `v2.3.0-alpha.1` is out? Got it.

Comment (MaximAlien) [reply README.md:106]: I think for major versions we were temporarily removing this part, but if `v2.3.0-alpha.1` is going to be out soon I guess it's fine to leave it.

**Pull request merged**